socorro/cron/jobs/archivescraper.py

```diff
@@ -298,14 +298,19 @@ def scrape_candidates(self, product_name, archive_directory):
         ]
 
         # If we've got a major_version, then we only want to scrape data for versions
-        # greater than (major_version - 3)
+        # greater than (major_version - 4) and esr builds
         if major_version:
-            major_version_minus_3 = major_version - 3
-            self.config.logger.info('Skipping anything before %s', major_version_minus_3)
+            major_version_minus_4 = major_version - 4
+            self.config.logger.info(
+                'Skipping anything before %s and not esr', major_version_minus_4
+            )
             version_links = [
                 link for link in version_links
-                # "63.0b7-candidates/" -> 63
-                if int(link['text'].split('.')[0]) >= major_version_minus_3
+                if (
+                    # "63.0b7-candidates/" -> 63
+                    int(link['text'].split('.')[0]) >= major_version_minus_4 or
+                    'esr' in link['text']
```

Comment:
This will pick up *all* esr builds even ones that we're not doing updates on anymore. We can probably optimize that out, but it'd require another query to the `product_versions` table. Right now, it's not a big deal--there aren't that many additional directories to scrape. It'll be more onerous over time.

Having said that, the incremental archivescraper pass takes under 3 minutes on my laptop, so I think this is fine.